src/test/kotlin/com/gamingbacklog/api/gamingbacklogapi/unit/controllers/LibraryControllerTests.kt

```diff
@@ -198,13 +199,13 @@ class LibraryControllerTests {
     @Test
     fun shouldSuccessfullyDeleteGameFromLibrary() {
       val library = Library("library-id1", "Backlog", arrayListOf("id1", "id2", "id3"))
-      given(libraryService.deleteGameFromLibrary("library-id1", "id1")).will {
+      given(libraryService.deleteGameFromLibrary(id2, id1)).will {
         library.games.remove("id1")
+        library
       }
-      endpoint += "library-id1/games/id1"
-      requestBuilder.runDeleteRequest(endpoint)
+      val request = UpdateLibraryGamesRequest(id1)
+      requestBuilder.runDeleteRequest("$endpoint$id2/games", Gson().toJson(request))
         .andExpect(status().isOk)
-        .andExpect(content().string(containsString("Successfully deleted game id1 from library library-id")))
```

Comment:
Should we now be expecting the library in the response here?

Reply:
yes good point!